chain/chain/src/runtime/tests.rs

```diff
@@ -369,8 +374,12 @@ impl TestEnv {
     }
 
     pub fn view_account(&self, account_id: &AccountId) -> AccountView {
-        let shard_id =
-            self.epoch_manager.account_id_to_shard_id(account_id, &self.head.epoch_id).unwrap();
+        let shard_id = EpochInfoProvider::account_id_to_shard_id(
+            self.epoch_manager.as_ref(),
```

Comment:
Yeah I can remove the `as_ref()` but I think I need to go for `&*` instead of just `&` because of

```
the trait bound `std::sync::Arc<near_epoch_manager::EpochManagerHandle>: near_primitives::types::EpochInfoProvider` is not satisfied
the following other types implement trait `near_primitives::types::EpochInfoProvider`:
  near_epoch_manager::EpochManagerHandle
```